scripts/tests/test_zkai_d64_rmsnorm_swiglu_statement_fixture.py

```
@@ -0,0 +1,130 @@
+        mutations = payload["mutation_suite"]
+        self.assertTrue(mutations["baseline_valid"])
+        self.assertEqual(mutations["decision"], "GO")
+        self.assertEqual(mutations["mutations_checked"], 14)
+        self.assertEqual(mutations["mutations_rejected"], 14)
+
+    def test_statement_binding_rejects_valid_looking_wrong_commitments(self) -> None:
+        statement = FIXTURE.build_fixture()["statement"]
+        mutated = copy.deepcopy(statement)
+        mutated["weight_commitment"] = "11" * 32
+
+        with self.assertRaisesRegex(FIXTURE.StatementFixtureError, "weight_commitment"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)